code/game/objects/items/storage/firstaid.dm

```diff
@@ -27,8 +27,13 @@
 		/obj/item/tool/surgery/surgical_line,
```

Comment:
- should (possibly) be able to hold generic surgical tools, they're part of medical and have good use
- should probably be able to hold surgical kits. It can already hold the fairly similar syringe kits, and the inability to hold a surgical kit means that using a knife will be fully preferable to the surgical kit HMs start with
- should be able to hold stasis bags. They're small and an invaluable thing to have on the front lines, with how many hugger traps can be around.

Reply:
Definite no on surgical tools
Surgical kit is probably fine